smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/KafkaConnector.java

```diff
@@ -90,6 +90,7 @@
 @ConnectorAttribute(name = "graceful-shutdown", type = "boolean", direction = Direction.INCOMING, description = "Whether or not a graceful shutdown should be attempted when the application terminates.", defaultValue = "true")
 @ConnectorAttribute(name = "poll-timeout", type = "int", direction = Direction.INCOMING, description = "The polling timeout in milliseconds. When polling records, the poll will wait at most that duration before returning records. Default is 1000ms", defaultValue = "1000")
 @ConnectorAttribute(name = "pause-if-no-requests", type = "boolean", direction = Direction.INCOMING, description = "Whether the polling must be paused when the application does not request items and resume when it does. This allows implementing back-pressure based on the application capacity. Note that polling is not stopped, but will not retrieve any records when paused.", defaultValue = "true")
+@ConnectorAttribute(name = "max.poll.buffer.size", type = "int", direction = Direction.INCOMING, description = "The maximum number of polled messages to buffer waiting to be consumed. This is used to limit the memory consumption while implementing back-pressure with `pause-if-no-requests`. Defaults to `max.poll.records` times 5.")
```

Comment:
I think the naming convention is `foo.bar.baz` for Kafka properties, and `foo-bar-baz` for "our" properties. So this should IMHO be `max-poll-buffer-size`.

Reply:
nice catch